respy/pre_processing/model_processing.py

```diff
@@ -150,6 +151,45 @@ def _parse_parameters(params, options):
     return optim_paras
 
 
+def _parse_present_bias_parameter(optim_paras, params):
+    """Parse present-bias parameter which is 1 by default.
+
+    Examples
+    --------
+    An example where present-bias parameter is specified:
+
+    >>> tuples = [("delta", "delta"), ("beta", "beta")]
+    >>> index = pd.MultiIndex.from_tuples(tuples, names=["category", "name"])
+    >>> data = [0.95, 0.4]
+    >>> params = pd.Series(data=data, index=index)
+    >>> optim_paras = {}
+    >>> _parse_present_bias_parameter(optim_paras, params)
+    {'beta': 0.4, 'beta_delta': 0.38}
+
+    And one where present-bias parameter is not specified:
+
+    >>> tuples = [("delta", "delta")]
+    >>> index = pd.MultiIndex.from_tuples(tuples, names=["category", "name"])
+    >>> data = 0.95
+    >>> params = pd.Series(data=data, index=index)
+    >>> optim_paras = {}
+    >>> _parse_present_bias_parameter(optim_paras, params)
+    {'beta': 1, 'beta_delta': 0.95}
+
+    """
+    if "beta" in params.index.get_level_values("category"):
+        beta = params.loc[("beta", "beta")]
+    else:
+        beta = 1
```

Comment:
Need trick for dictionaries, but also for DataFrames and Series.
```suggestion
    beta = params.get(("beta", "beta"), 1)
```